### DISCUSSION TITLE
.NET 7 Release

### DISCUSSION THREAD

**Discussion by BinaryAssault** [enisn/UraniumUI#101] [Q&A]
Hi @enisn, do you have an estimated time for .NET7 compatibility release?

**Comment by enisn** [comment]
Sorry for waiting too long. A preview version has been released as `2.0.0-pre.1`. There are some issues, after resolution of those issues, I'll release a stable version for 2.0

**Comment by BinaryAssault** [comment]
Thank you, do you take pull requests? I don't mind helping if I have time. 

Edit - haven't done much testing but the initial crashing on Init is no longer present. Thanks for all the work on this

**Reply by enisn** [reply]
Yes PR is always welcome, you can start with one of issue that has [good first issue](https://github.com/enisn/UraniumUI/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) label or you can just create an issue about your proposal about what will you change for discussion
